shanoir-uploader/pom.xml

```diff
@@ -22,6 +22,8 @@
 		<junit.jupiter.version>5.8.2</junit.jupiter.version>
 		<junit.platform.version>1.8.2</junit.platform.version>
 		<poi.version>4.1.2</poi.version>
+		<maven.build.timestamp.format>yyyy-MM-dd'T'HH-mm-ss'Z'</maven.build.timestamp.format>
```

Comment:
Is it part of the correction ?

Reply:
No, it's so shanoiruploader builds on windows (can't have path with ":" in it). I'll take this off